Editor/HPUIInteractorConeRayAnglesEditor.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEditor;
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine.XR.Hands;
+
+namespace ubco.ovilab.HPUI.Editor
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIInteractorConeRayAngles), true)]
+    public class HPUIInteractorConeAnglesEditor: UnityEditor.Editor
+    {
+        private HPUIInteractorConeRayAngles t;
+
+        private float offsetValue;
+        private bool applyToAll;
+        private bool overwriteThisAsset;
+        private XRHandJointID selectedPhalange = XRHandJointID.IndexDistal;
+
+        // storing all baseline thresholds for first ray per phalange
+        // assuming that offsets are applied uniformly per phalange
+        private Dictionary<XRHandJointID, float> baselineThresholds = new Dictionary<XRHandJointID, float>();
+        // storing previous applied offset
+        private Dictionary<XRHandJointID, float> cachedOffsets = new Dictionary<XRHandJointID, float>();
+
+        private const string DONT_ASK_FOR_OVERRIDE_EDITOR_PREF_KEY = "ubco.ovilab.HPUI.Components.ConeThresholdsEditor.DontAskForOverride";
```

Comment:
make this "ubco.ovilab.HPUI.Interaction. ......."

Reply:
I used the [EstimateConeRayAngles](https://github.com/ovi-lab/HPUI-Core/blob/development/Editor/EstimateConeRayAnglesEditor.cs#L24) script
```cs
private const string DONT_ASK_EDITORPREF_KEY = "ubco.ovilab.HPUI.Components.ConeEsimation.DontAskWhenRestarting";
```
as a reference. Not sure what the difference is with that and this?

Reply:
also since it's in session, it doesn't save it as a registry key, it's only within Unity as a cached session parameter